docs/onboard-public-officers-using-non-se-machines.md

```diff
@@ -1,37 +1,109 @@
+[SEED](https://docs.developer.tech.gov.sg/docs/security-suite-for-engineering-endpoint-devices/#/)-->
+
+You can use the [TechPass portal](http://portal.techpass.gov.sg) to do a self-service sign-up for a TechPass account, or another public officer can invite you to TechPass via the [TechBiz Portal](portal.techbiz.suite.gov.sg). For more information, refer to [TechBiz documentation](https://docs.developer.tech.gov.sg/docs/techbiz-documentation/).
+
+## Audience
+
+- Public officers using a non-SE GSIB device with a WOG ID. WOG ID is the organisational email address belonging to the *gov.sg* domain and is in the format of *your_name<span>@</span><acronym for your agency>.gov.sg*.
+
+- Vendors or contractors using a non-SE GSIB device and whose organisational email address is in the format of *your_name_from.<vendor organisation name>@<acronym for the agency>.gov.sg*.
+
+> **Note**: If you are a public officer using a SE-GSIB device and intend to request for TechPass or SEED, create a [service request](https://go.gov.sg/techpass-sr) with us. Alternatively, another public officer can invite you to TechPass via the [TechBiz Portal](portal.techbiz.suite.gov.sg).
+
+## Prerequisites
+
+- If you are going to sign up for TechPass account via the TechPass portal, you need a non-SE GSIB device.
+- To onboard to TechPass, you must have received the TechPass onboarding email.
+- Standard mailbox for the users' organisational email addresses. TechPass does not support email accounts which do not have an inbox, such as LiteMail accounts. If you use such an email account, upgrade it to a standard mailbox before signing up for TechPass.
+
+> **Note**: You will receive the TechPass onboarding email when you sign up for TechPass from the TechPass Portal, or a public officer invites you to TechPass from the [TechBiz Portal](portal.techbiz.suite.gov.sg).
+
 
-If you are a public officer, complete the following steps to get a TechPass account and [SEED](https://docs.developer.tech.gov.sg/docs/security-suite-for-engineering-endpoint-devices/#/) provisioning. You need SEED provisioning to access SGTS services from your internet device.
 
 > **Tip**: Click the triangle to view the instructions to complete each step.
 
 
-## Step 1. Set up security verification for your WOG account
+## Step 1. Sign up for TechPass and SEED
+
+<details>
+  <summary style="font-size:20px;font-weight:bold">How to sign up for TechPass?</summary>
+
+  **To get TechPass invitation email**
+
+  1. From your non-SE GSIB device, go to the [TechPass portal](http://portal.techpass.gov.sg) and click **Sign Up**.
+
+  <kbd>![sign-up](assets/images/onboarding/po-non-se/sign-up-new.png)</kbd>
+
+  2. Enter your organisational email address.
+
+  > **Tip**: For more information on the acceptable email formats, refer to the [Audience](#audience) section.
+
+  3. Indicate if you want to onboard your non-GSIB or non-DWP device to SEED and select **I'm not a robot**.
```

Comment:
just curious ah.. why do we mention dwp device here ah?

we don't support dwp devices right?

Reply:
It was stated first in Onboard to SEED section, so i continued using it :-).

Reply:
Please let me know if we should not mention about DWP :-)